tests/suites/test_suite_cipher.chacha20.data

```diff
@@ -1,3 +1,7 @@
+ChaCha20 Decrypt empty buffer
```

Comment:
ChaCha20 is a stream cipher, so the ciphertext and the plaintext have the same length, so this is already covered by "ChaCha20 Encrypt and decrypt 0 bytes".